[2024-12-02T06:33:03.186+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-02T06:33:03.209+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_data_elt.dbt_snapshot manual__2024-12-02T06:32:33.505492+00:00 [queued]>
[2024-12-02T06:33:03.215+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_data_elt.dbt_snapshot manual__2024-12-02T06:32:33.505492+00:00 [queued]>
[2024-12-02T06:33:03.215+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-02T06:33:03.225+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_snapshot> on 2024-12-02 06:32:33.505492+00:00
[2024-12-02T06:33:03.232+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_data_elt', 'dbt_snapshot', 'manual__2024-12-02T06:32:33.505492+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/data_elt.py', '--cfg-path', '/tmp/tmppdmkz4om']
[2024-12-02T06:33:03.233+0000] {standard_task_runner.py:91} INFO - Job 78: Subtask dbt_snapshot
[2024-12-02T06:33:03.234+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1520) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-02T06:33:03.234+0000] {standard_task_runner.py:63} INFO - Started process 1521 to run task
[2024-12-02T06:33:03.264+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_data_elt.dbt_snapshot manual__2024-12-02T06:32:33.505492+00:00 [running]> on host 065cc2990949
[2024-12-02T06:33:03.327+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_data_elt' AIRFLOW_CTX_TASK_ID='dbt_snapshot' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T06:32:33.505492+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T06:32:33.505492+00:00'
[2024-12-02T06:33:03.328+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-02T06:33:03.338+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-02T06:33:03.339+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt snapshot --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2024-12-02T06:33:03.346+0000] {subprocess.py:86} INFO - Output:
[2024-12-02T06:33:04.975+0000] {subprocess.py:93} INFO - [0m06:33:04  Running with dbt=1.8.7
[2024-12-02T06:33:05.807+0000] {subprocess.py:93} INFO - [0m06:33:05  Registered adapter: snowflake=1.8.4
[2024-12-02T06:33:06.771+0000] {subprocess.py:93} INFO - [0m06:33:06  Found 4 models, 11 data tests, 1 source, 575 macros
[2024-12-02T06:33:06.775+0000] {subprocess.py:93} INFO - [0m06:33:06
[2024-12-02T06:33:06.776+0000] {subprocess.py:93} INFO - [0m06:33:06  Nothing to do. Try checking your model configs and model specification args
[2024-12-02T06:33:07.526+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-02T06:33:07.527+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-02T06:33:07.546+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nba_data_elt, task_id=dbt_snapshot, run_id=manual__2024-12-02T06:32:33.505492+00:00, execution_date=20241202T063233, start_date=20241202T063303, end_date=20241202T063307
[2024-12-02T06:33:07.577+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-02T06:33:07.588+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-02T06:33:07.591+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
